default.nix

```diff
@@ -77,6 +41,27 @@ rec {
       in
       f "";
 
+    # Recursively evaluate attributes for an attribute set.
+    # Coupled with an evaluated nixos configuration, this presents an efficient
+    # way for checking module types.
+    forceEvalRecursive =
```

Comment:
Won't this always succeed evaluating, even if there's a type error in the module? IIUC the check will produce a drv with true/false in an attribute tree, so that's not where it could "break" either.

Reply:
If type-checking fails then the evaluation will fail with an error and not `false`:

```nix
  extensions = true;
  links = { };
  module = «error: The option `projects.Omnom.nixos.modules.services.omnom.module' was accessed but has no value defined. Try setting the option.»;
  name = true;
```

This will thus cause the build to fail (see [more examples](https://github.com/ngi-nix/ngipkgs/pull/1003#issuecomment-2858195536)), so in a sense the attribute tree will never have `false` inside it, and that's okay for us since we only care about forcing evaluations.